editor/dependency_editor.cpp

```diff
@@ -531,6 +531,28 @@ void DependencyRemoveDialog::_build_removed_dependency_tree(const Vector<Removed
 	}
 }
 
+static String get_file_name(const Vector<String> &dirs_to_delete, const Vector<String> &files_to_delete) {
```

Comment:
```suggestion
static String get_to_delete_string(const Vector<String> &dirs_to_delete, const Vector<String> &files_to_delete) {
```
More descriptive naming IMO, as it's not a single file, or even just files 🙂

Reply:
Thanks! I decide to use itemList and change this function lol.